WooCommerce/WooCommerceTests/ViewRelated/Survey/URL+SurveyViewControllerTests.swift

```diff
@@ -0,0 +1,49 @@
+import Foundation
+import XCTest
+
+@testable import WooCommerce
+
+/// Test cases for `URL extension at SurveyViewController`.
+
+final class URL_SurveyViewControllerTests: XCTestCase {
+
+    func test_tag_platform_append_the_correct_tag_data() throws {
```

Comment:
super nit: (same for the other test cases)

```suggestion
    func test_tagging_platform_appends_the_correct_tag_data() throws {
```

Reply:
Oops, thanks for catching those typos, Jaclyn! I've submitted it right here: https://github.com/woocommerce/woocommerce-ios/pull/3041/commits/703ab9213616143cac95e38e29001c9053f73f52